docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
+
+.. task:: enable_flocker_control
```

Comment:
Because of https://github.com/sbrunner/sphinx-prompt/issues/2 this is broken in at least Safari and Chrome. That issue did not affect us much because previously it was only AFAIK a minor problem in the release process docs. See the screenshot:
![screenshot 2015-03-16 14 52 16](https://cloud.githubusercontent.com/assets/797801/6668739/ba96e1ce-cbec-11e4-8af4-a3274659180d.png)

This really confused me reading the rendered docs. That tells me that there should be prose explaining where these commands are run. There are a few options:
* Live with the rendering problem and hope that upstream fixes it. I don't like this option.
* Not use `sphinx-prompt` here.
* Submit a fix upstream.